\documentclass[11pt]{article}


\usepackage[margin=1.0in]{geometry}
\usepackage{tablefootnote}
\usepackage{graphicx}
\usepackage{natbib}
\usepackage{amsmath}
\usepackage{color}



\title{Current data show no signal of Ebola virus adapting to humans}

\author{Stephanie J. Spielman$^{1^\ast}$, Austin G. Meyer$^{1,2^\ast}$, and Claus O. Wilke$^{1}$}


\begin{document}
\maketitle
\maketitle
\noindent
Address:\\
$^1$Department of Integrative Biology, Center for Computational Biology and Bioinformatics, and Institute of Cellular and Molecular Biology.
The University of Texas at Austin, Austin, TX 78712, USA.\\
$^2$ School of Medicine, Texas Tech University Health Sciences Center, Lubbock, TX 79430, USA \\
$^\ast$Authors contributed equally to this work.


\begin{abstract}
{\color{blue}Gire et al.\ (\emph{Science} 345:1369--1372, 2014) analyzed 81 complete genomes sampled from the 2014 Zaire ebolavirus (EBOV) outbreak and claimed that the virus is evolving far more rapidly in the current outbreak than it has been between previous outbreaks. This assertion has received widespread attention, and many have perceived \citet{Gire2014}'s results as implying rapid adaptation of EBOV to humans during the current outbreak. Here, we show that, on the contrary, sequence divergence in EBOV is rather limited, and that the currently available data contain no signal of rapid evolution or adaptation to humans. Gire et al.'s findings resulted from an incorrect application of a molecular-clock model to a population of sequences with minimal divergence and segregating polymorphisms. Our results highlight how indiscriminate use of off-the-shelf analysis techniques may result in highly-publicized, misleading statements about an ongoing public health crisis.}
\end{abstract}

\noindent
Zaire ebolavirus (EBOV) is currently devastating West African populations in an unprecedented epidemic that has begun to spill over into many parts of the world. Since its discovery in the 1970s, EBOV has, at regular intervals, caused zoonotic outbreaks in human populations. Unlike past outbreaks, however, the current EBOV outbreak shows sustained transmission among humans, prompting concerns that as the outbreak escalates, EBOV may evolve to become endemic in humans. Recently, \citet{Gire2014} published 99 genomes from 78 patients infected in the current outbreak, sampled during May and June of 2014 in Sierra Leone. They analyzed these 78 of these genomes (one for each patient), in combination with three EBOV genomes collected in Guinea during March 2014 \citep{Baizeetal2014}, and reported ``a rapid accumulation of...genetic variation.'' They additionally stated that ``[t]he observed substitution rate is roughly twice as high within the 2014 outbreak as between outbreaks.'' The conclusions ultimately left readers, and indeed the scientific community at large, with the impression that EBOV is fast-evolving and possibly adapting to humans (see also \citealt{NatureEditorial,Lukszaetal2014,Alexanderetal2014}). 

By contrast, we do not find any robust evidence in the available 2014-outbreak EBOV genomes supporting this interpretation. While it is clear that mutations are certainly occurring in EBOV, the available genomic data do not show concrete evidence that EBOV is evolving rapidly. In fact, among the 81 genomes from the current outbreak, there are only 29 unique sequences (two from Guinea and 27 from Sierra Leone), and no genome contains more than 2 nonsynonymous mutations.

To put EBOV's evolutionary dynamics into context, we compared the extent of genetic diversity within 2014-outbreak EBOV genes to the genetic diversity accumulated during the early months of the 2009 pandemic H1N1 influenza outbreak. Influenza virus is the archetypal rapidly-adapting human virus, and, like EBOV, it is a negative-sense single-stranded RNA virus. Since 1977, the two circulating seasonal influenza A strains have been H1N1 and H3N2. In late 2008 or early 2009, a new H1N1 strain, pandemic H1N1, a reassortant of several influenza viruses circulating in swine, was first transmitted from swine to humans and caused the 2009 H1N1 pandemic \cite{Smithetal2009}. The first sample of pandemic H1N1 was identified on April 15, 2009 \citep{Neumannetal2009}. We considered here only H1N1 sequences collected in April 2009. 

We constructed gene trees for EBOV nucleoprotein (\emph{np}) and polymerase (\emph{l}), selecting only genes from the 2014-outbreak EBOV sequences (Figure~\ref{ebov_flu}A). \emph{np} and \emph{l} have accumulated the most sequence diversity of all seven EBOV genes during this outbreak, and thus they likely represent the most rapidly-evolving EBOV genes. We constrasted these phylogenies with gene trees for H1N1 hemagglutinin (\emph{HA}) and nucleoprotein (\emph{NP}) (Figure~\ref{ebov_flu}B) for sequences collected within a single month (April 2009) in the US. {\color{blue}\emph{HA}, the influenza surface protein responsible for host receptor-binding, is under intense selection pressure to evade host immunity, and it experiences elevated rates of nonsynonymous evolution near its receptor-binding site, though less so in H1N1 than in the more rapidly evolving H3N2 \citep{Meyer2012}. \emph{NP} is not exposed on the viral envelope and evolves relatively slowly \textbf{we need an appropriate H1N1 reference here.}
%, with only 0.6\% sites under positive selection \citep{Linetal2011}.
}

As the juxtaposed gene trees in Figures~\ref{ebov_flu}A and~\ref{ebov_flu}B  demonstrate, even the fastest-evolving EBOV sequences from the 2014 outbreak are evolving much more slowly than H1N1 sequences do over similar temporal scale. Indeed, even the slowly-evolving influenza \emph{NP} far outpaces EBOV genes in terms of accumulated genetic diversity. Furthermore, the average nucleotide diversities among the EBOV genes \emph{np} and \emph{l} from the current outbreak are approximately an order magnitude lower than the nucleotide diversities observed for the H1N1 genes \emph{HA} and \emph{NP}, while the mean root-to-tip distances are between 3 and 7 times lower in the EBOV genes than in the H1N1 genes (Table~\ref{distances}). Thus, EBOV is simply not accumulating mutations in a manner we would expect from a rapidly evolving virus.

In fact, even considering all EBOV outbreaks since 1976, we find limited evidence for evolutionary divergence. For example, the mean root-to-tip distances in EBOV genes over all outbreaks (spanning four decades) are comparable to those found in influenza H1N1 virus in a single month (Table~\ref{distances}). Figure~\ref{ebov_flu}C shows a phylogeny of all EBOV sequences considered in \citet{Gire2014}, as well as related ebolavirus species. The extent of sequence divergence across EBOV sequences collected from 1976 to present pales in comparison to divergence among ebolavirus species. The minimal sequence divergence among, and within, EBOV outbreaks indicates that the current EBOV outbreak should be considered a single, polymorphic population. Indeed, the mean pairwise sequence similarity among \emph{unique} 2014-outbreak EBOV genomes is 99.84\% (standard deviation of 0.39\%). Moreover, according to \citet{Gire2014}, there are least 55 segregating mutations in the 2014-outbreak EBOV sequences, yet these mutations generally do not co-occur in any particular genome. For example, among these EBOV genome sequences, only three (two from Sierra Leone and one from Guinea) contain two nonsynonymous mutations, no genome contains three or more nonsynonymous changes, and there is no evidence that any given site has experienced multiple mutation events in the current outbreak.  

{\color{blue}Taken together, these results reveal that EBOV sequence data must be analyzed in a population genetics, rather than a purely phylogenetic, context. However, \citet{Gire2014} did not do so. For example, \citet{Gire2014} found that the 2014-outbreak EBOV substitution rate is elevated relative to a baseline EBOV substitution rate, measured from pooling all sequence data collected since 1976 (Figure 4F in \citealt{Gire2014}), by inferring the rate of a strict molecular clock. Given the minimal divergence of the 2014 sequences, this approach is fundamentally flawed. The molecular clock assumes that sequences have sufficiently diverged such that all differences are fixed substitutions rather than segregating polymorphisms. As a consequence, the rate of the molecular clock is highly time-dependent, such that the substitution rate is substantially elevated at short time-scales due to the confounding presence of segregating polymorphisms \citep{Hoetal2005, Hoetal2007, PetersonMasel2009, Hoetal2011}. \dots but failed to mention that this lack of purifying selection in effect invalidates the substitution-rate calculation. Indeed, other groups have simply quoted the doubled substitution rate without qualification regarding its origin \citep{Alexanderetal2014}. 
% Thus, that \citet{Gire2014} observed an elevated substitution rate within the 2014-outbreak sequences is actually an expected outcome and does not provide any evidence for accelerated evolution. Instead, it is merely a signal of the minuscule divergence among these sequences.
Until more time has passed and mutations have either fixed or been removed from the population, results concerning EBOV substitution rate in the current outbreak are unreliable and inconclusive, and should clearly be labeled as such.
}

{\color{blue}In sum, we do not find any convincing evidence in \citet{Gire2014}'s study supporting their conclusion that EBOV 2014 sequences are rapidly evolving. As more sequence data from the current outbreak are collected and made available for analysis, evidence may emerge to support these claims. However, until such data are released, we cannot conclude that EBOV 2014 sequences show signatures of increased evolutionary rate, much less of adaptation to humans.}

\section*{Acknowledgements}
This work was supported in part by DTRA grant HDTRA1-12-C-0007 and NSF Cooperative Agreement No.\ DBI-0939454 (BEACON Center). Computational resources were provided by the University of Texas at Austin's Center for Computational Biology and Bioinformatics (CCBB).




\bibliographystyle{MBE}
\bibliography{references}
\clearpage

\begin{table}
\caption{\label{distances}Mean root-to-tip distance and nucleotide diversity for EBOV and influenza sequences.}
\bigskip

\centerline{
\begin{tabular}{ccccc}
\hline
\textbf{virus} & \textbf{outbreak} & \textbf{gene} & \textbf{mean root-to-tip distance} & \textbf{nucleotide diversity} \\
\hline
EBOV & 2014 & \emph{np}  &  0.0010   & 0.00025  \\
 & & \emph{l} &  0.0019  & 0.00016  \\
 & all (1976-2014)& \emph{np} & 0.0078 & 0.0097~\, \\
 &  & \emph{l}  & 0.0080   & 0.0087~\, \\
H1N1 & April 2009 & \emph{HA} & 0.0071   & 0.0018~\, \\
 &  & \emph{NP}   &  0.0053 & 0.0018~\, \\
\hline
\end{tabular}
}
\end{table}

\clearpage
\begin{figure}[H]
	\centerline{\includegraphics[width=14cm]{ebov_trees_H1N1.pdf}}
	\caption{\label{ebov_flu} Limited divergence in EBOV 2014. Phylogenies in A and B were constructed from nucleotide data in FastTree2 \citep{FTT2} under the GTR model. Sequences in A were restricted to 2014-outbreak EBOV sequences, and the phylogenies were rooted on \emph{np} and \emph{l} sequences sampled in 2002. Sequences in B were collected from the Influenza Research Database \citep{ird}, with the search restricted to complete segments of pandemic H1N1 sampled during April 2009 in the US. Redundant strains, lab strains, and seasonal H1N1 were excluded. H1N1 phylogenies were rooted on the \emph{HA} and \emph{NP} genes from H1N1 strain A/Memphis/15/2000. The phylogeny in C was constructed from the genomic data of \citet{Gire2014} for all ebolavirus species, using RAxMLv.8.1.1 \citep{raxml} with the GTR+GAMMA model and a different partition for each gene. Numbers at nodes indicate bootstrap support. Abbreviations shown in the figure stand for Reston virus (RESTV), Bundibugyo virus (BDVD), Ta\"{i} forest virus (TAFV), and Sudan virus (SUDV). 
	}
\end{figure}

\end{document}

Reported substitution rates:
pH1N1: 3.66e-3 (Smith et al. 2009) (but just for 2009 outbreak)
EBOV: 8e-4 (Gire2014) (all of EBOV, not 2014 outbreak)
H3N2: 5.83e-3 (Bedford et al. 2010)
